### ISSUE TITLE
Online PO Editor - Need to find a replacement

### ISSUE THREAD

**Issue by jwalantsoneji** [duckduckgo/duckduckgo-locales#227]
Hello,

DuckDuckGo locales has mention of Online PO Editor on its main page in GitHub.
Tried to open Gujarati language PO file in it and it said the file was larger than it could accept; the allowed max size is 1MB.
Now, Gujarati is not even fully translated yet. So, it should be a problem with other locales too.
Please check and find replacement online editor for PO files if possible.

**Comment by soumendrak** [comment]
@jwalantsoneji , Could you please point us to the file you are trying to upload?
In most of the languages the files are less than 300KB size.

**Comment by jwalantsoneji** [comment]
> The [Gujarati PO file](https://github.com/duckduckgo/duckduckgo-locales/blob/master/locales/gu_IN/LC_MESSAGES/duckduckgo.po) size is 103KB only.

As I am downloaded the file right now and checking its size in Windows 10. It is above 1MB --> 1,234 KB to be precise. And still online PO editor poedit.net is declining to open due to hitting over size.
![image](https://user-images.githubusercontent.com/626292/72001663-ea9ac380-3213-11ea-8eca-4a2f9b65b587.png)

**Comment by jbarrett** [comment]
Hi @jwalantsoneji. Are you using the browser save function on [this page](https://github.com/duckduckgo/duckduckgo-locales/blob/master/locales/gu_IN/LC_MESSAGES/duckduckgo.po)? That will save the PO file plus GitHub's HTML which is indeed ~1.2MB.

The [raw file is here](https://raw.githubusercontent.com/duckduckgo/duckduckgo-locales/master/locales/gu_IN/LC_MESSAGES/duckduckgo.po)

Thanks!

**Comment by jwalantsoneji** [comment]
> Hi @jwalantsoneji. Are you using the browser save function on [this page](https://github.com/duckduckgo/duckduckgo-locales/blob/master/locales/gu_IN/LC_MESSAGES/duckduckgo.po)? That will save the PO file plus GitHub's HTML which is indeed ~1.2MB.
> 
> The [raw file is here](https://raw.githubusercontent.com/duckduckgo/duckduckgo-locales/master/locales/gu_IN/LC_MESSAGES/duckduckgo.po)
> 
> Thanks!

Yes, I was right clicking on the .po file on the first link above and clicking Save As from the context menu.
The second link which is to the raw file, is all good. Thanks! How to get to this link? Can we please add that to the local's page so that it's easy for all?

**Comment by jwalantsoneji** [comment]
Please note that the raw file has different header items compared to the file inside master.
When I am done with changes, do I need to copy only the changed lines? Replacing complete file is easier.

**Comment by Madis0** [comment]
You do replace the whole file, Git is smart enough to see what has changed.
